### ISSUE TITLE
Jetpack App (Basics): Handle Magic Links

### ISSUE THREAD

**Issue by zwarm** [wordpress-mobile/WordPress-Android#14620]
Parent #14293 

- [x] Update magic-link scheme for Jetpack app. @ashiagr #14644
- [x] Handle no site scenario for login via magic link. @ashiagr  #14662

**Comment by leandroalonso** [comment]
JFYI: when requesting the magic link email the `scheme` property should be `jetpack`. Ie.:

```
curl https://public-api.wordpress.com/rest/v1.1/auth/send-login-email -H 'Content-Type: application/json' -d '{ "email": "$YOUR_WPCOM_EMAIL_HERE$", "client_id": $YOUR_CLIENT_ID$, "client_secret": "$YOUR_CLIENT_SECRET$", "scheme": "jetpack" }'
```

**Comment by ashiagr** [comment]
Closed via #14662
